recipes/nucleoatac/meta.yaml

```diff
@@ -28,15 +28,12 @@ requirements:
     - scipy
     - matplotlib
     - pysam >=0.8.1
+    - zlib
   run:
     - python
     - cython >=0.22
     - numpy >=1.9.1
     - scipy
     - matplotlib
     - pysam >=0.8.1
-    
-test:
-  commands:
-    - nucleoatac run --help
-    - pyatac --help
\ No newline at end of file
+    - zlib
```

Comment:
This looks also wrong. Why zlib here? It should be a run dependency, but giving that the test pass?

Reply:
Sorry, I'm not sure I follow. Yout think it should, or should not, be a run dependency? My issue with this package was that is builds some cython code when you attempt to import, for example, `pyatac.fragmentsizes`. This compilation was failing under the old version and wasn't tested. My intention here was to ensure the test actually does something other than display help (which doesn't properly test dependencies). Really, this should probably be done during `setup.py`, but that's an upstream thing I didn't want to get into just now.

Reply:
Oh, sorry, I just saw this wrong on my mobile. You have actually removed the test commands here. Can you please get them back :)
Also please pin zlib as well.
Thanks @lparsons!